Formula/font-util.rb

```diff
@@ -29,6 +30,8 @@ def install
     system "./configure", *args
     system "make"
     system "make", "install"
+    # avoid cellar in shared fonts dir
+    inreplace lib/"pkgconfig/fontutil.pc", prefix, HOMEBREW_PREFIX
```

Comment:
This seems better since it allows for unlinked software
```suggestion
    inreplace lib/"pkgconfig/fontutil.pc", prefix, opt_prefix
```

Reply:
Variables `fontrootdir` in `fontutil.pc` is used to determine the default location of X11 fonts, see [meson.build](https://gitlab.freedesktop.org/xorg/xserver/-/blob/master/meson.build#L151), this path should be shared by [multiple formulae](https://www.x.org/releases/individual/font/) in future, so `HOMEBREW_PREFIX/share/fonts/X11` is more reasonable. If modify `prefix` is not good, we can just modify `fontrootdir` or `datarootdir`.

Reply:
Patch only `datarootdir` now.